src/gameStateService/game-state-service.ts

```diff
@@ -171,6 +184,17 @@ export class GameStateService extends PubSub {
     getStonePositions() {
         return createStonePositionsDAO(this);
     }
+    
+    levelEndSceneData({levelEndData, data}) {
+        this.levelEndData = {...levelEndData};
```

Comment:
Is there a need to store scene data in the game state?

Check if:
- it is shared across different components
- only used to compute state